arrows/qt/widgets/MetadataView.cxx

```diff
@@ -0,0 +1,325 @@
+/*ckwg +29
```

Comment:
I haven't gotten around to changing anything yet, but maybe now is a nice opportunity to start using the new copyright banner?
```
// This file is part of KWIVER, and is distributed under the
// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
```